priv/cabbage/apps/itest/lib/reorg.ex

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Itest.Reorg do
+  @moduledoc """
+    Chain reorg triggering logic.
+  """
+  use GenServer
+
+  require Logger
+
+  @node1 "geth-1"
+  @node2 "geth-2"
+  @pause_seconds 100
+
+  def finish_reorg() do
+    if Application.get_env(:cabbage, :reorg) do
+      unpause_container!(@node1)
+      unpause_container!(@node2)
+
+      Process.sleep(60 * 1000)
```

Comment:
where does this number come from?

Reply:
After reorg finishes, nodes need time to connect to each other, it doesn't happen instantly. I set this value based on failing tests

Reply:
Perhaps we can figure out how to make this more dynamic. Can we detect this this somehow? Parse logs?

Reply:
I tried decreasing this number. Logs show that reorg happened, but the node still return 502